docs/queries/layout.md

```diff
@@ -20,12 +20,14 @@ parent: Queries
 
 ## Hiding Elements
```

Comment:
I wonder whether it is worth updating this heading now it's possible to show elements as well?

Reply:
Ok, done